### DISCUSSION TITLE
\par

### DISCUSSION THREAD

**Discussion by Matilde-rod** [joaomlourenco/novathesis#506] [Q&A] [answered]
Boa tarde :)
Nao consigo fazer o \par funcionar e queria paragrafos depois dos titulos de secçao :/ alguem sabe o que posso fazer?

**Comment by joaomlourenco** [comment]
Please use English in this forum.

A paragraph is a sequence of sentences which, in turn, are sequences of words which, in turn, are sequences of letters.

I'm sure you are expressing your wish in the wrong way!

What do you really want? Increase the space between the section header and the first line of text that follows? Or you want the first word of the first paragraph to be indented (advanced) similarly to the following paragraphs? Or something else?

**Reply by Matilde-rod** [reply]
Sorry :/
Yesyes, what i'm refereing to is the indentation of the paragraph.
All of the paragraphs have indentations execpt the first folowing each chapter/section, normally i use \par on overleaf for that but here i can't make it work

**Reply by joaomlourenco** [reply, marked as answer]
\par does not do what you believe it does.  \par is the TeX equivalent to leaving one empty line in LaTeX.

The solution for your problem is
```latex
\usepackage{indentfirst}
```
but notice that if you are writing in English, you **should not** indent the first word of the first paragraph (those are the rules for English).  If you are writing in Portuguese, there is no strict rule, and you may or may not indent the first word of the first paragraph.

**Reply by Matilde-rod** [reply]
Thank you for the explanation!
